SplunkRumWorkspace/SplunkRum/SplunkRum/OpenTelemetry/OpenTelemetrySdk/Trace/RecordEventsReadableSpan.swift

```diff
@@ -8,11 +8,23 @@ import Foundation
 /// Implementation for the Span class that records trace events.
 public class RecordEventsReadableSpan: ReadableSpan {
     public var isRecording = true
+
+    fileprivate let internalStatusQueue = DispatchQueue(label: "org.opentelemetry.RecordEventsReadableSpan.internalStatusQueue", attributes: .concurrent)
```

Comment:
Could the duplicate spans be coming from otel creating spans from different threads?  I am wondering if a serial queue would fix the problem.

Reply:
Was fixed with https://github.com/open-telemetry/opentelemetry-swift/pull/400